fs/rc/config.go

```diff
@@ -49,7 +49,7 @@ Parameters:
 
 Note that these are the global options which are unaffected by use of
 the _config and _filter parameters. If you wish to read the parameters
-set in _config then use options/config and for _filter use options/filter.
+set in _config then use options/local and for _filter use options/filter.
```

Comment:
I think the `options/filter` is wrong too.

`options/local` returns both `config` and `filter` settings.

So I think this should say something like

> If you wish to read the parameters set in `_config` or `_filter` use `options/local`.

What do you think?

Reply:
Okay 
fixed